map.js

```diff
@@ -284,4 +286,11 @@ map.withIndex = mapper => function mappingWithIndex(value) {
   throw new TypeError(`${value} is not an Array`)
 }
 
```

Comment:
The documentation at https://rubico.land/ starts right here in the comments. If you write comment documentation according to the following format, it all gets parsed into rubico.land via some scripts I set up.

**
 \* @name myFunc
 \*
 \* @synopsis
 \* \`\`\`coffeescript [specscript]
 \* myFunc(param1 any, param2 any) -> result any
 \* \`\`\`
 \*
 \* @description
 \* Hey, so below you can actually inline a code runner like this
 \* \`\`\`javascript [playground]
 \* console.log(
 \*   myFunc(1, 2),
 \* ) // something
 \* \`\`\`
 \*

Reply:
Fancy! I've updated the documentation here. Let me know if that requires any changes.